projects/gateway2/query/httproute.go

```diff
@@ -346,6 +348,17 @@ func (r *gatewayQueries) fetchChildRoutes(
 		// If the namespace is not explicitly set to a wildcard, restrict the List to the delegated namespace
 		if delegatedNs != wellknown.RouteDelegationLabelSelectorWildcardNamespace {
```

Comment:
where did we land on providing override for this?

Reply:
Which override are you referring to?